.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,67 @@
+{
+	"name": "hnp",
+	"build": {
+		"args": {
+			"ENERGYPLUS_VERSION": "9.5.0",
+			"ENERGYPLUS_INSTALL_VERSION": "9-5-0",
+			"ENERGYPLUS_SHA": "de239b2e5f"
+		}
+	},
+	"dockerFile": "../Dockerfile",
```

Comment:
Seems like this is using the Dockerfile in the parent dir, if so, maybe remove the Dockerfile in .devcontainer?

Reply:
Removed it now.